web_socket.example.env

```diff
@@ -0,0 +1,15 @@
+#copy-pasted from https://github.com/CodersCommunity/http-websocket-server .env example
```

Comment:
komentarz zbędny w tym miejscu

Reply:
Hmm, a może skoro zawartość tego pliku jest kopią z innego, to stworzyć wspólny plik (taki _Single source of truth_), żeby w razie przyszłych zmian nie musieć pamiętać o robieniu ich w dwóch osobnych miejscach?

Chociaż nie wiem czy technicznie da się to zrobić prościej niż przez [Git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules).